gravitee-am-reporter/gravitee-am-reporter-kafka/src/main/java/io/gravitee/am/reporter/kafka/audit/DtoMapper.java

```diff
@@ -82,7 +82,7 @@ public AuditEntityDto map(AuditEntity auditEntity) {
     dto.setId(auditEntity.getId());
     dto.setType(auditEntity.getType());
     dto.setReferenceId(auditEntity.getReferenceId());
-    dto.setReferenceType(this.map(auditEntity.getReferenceType()));
+    dto.setReferenceType(auditEntity.getReferenceType() != null ? this.map(auditEntity.getReferenceType()): null);
```

Comment:
The fix works but maybe we should do this test into the map method to be sure that other usage of this method using a null ReferenceType will not break the injestion of kafka message. wdyt ?

Reply:
done, moved the null check as suggested.